homeassistant/components/tuya/button.py

```diff
@@ -24,34 +24,79 @@
     "sd": (
         ButtonEntityDescription(
             key=DPCode.RESET_DUSTER_CLOTH,
+            action=DPCode.RESET_DUSTER_CLOTH,
```

Comment:
We don't need `action`, use `key`.

Reply:
I understand it will work the same way, but what I wanted to demonstrate is that it might be a good idea to use specific properties for the button entity.

Reply:
Sure, but implement it locally in an child class of ButtonEntityDescription.